src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfigTest.java

```diff
@@ -6,10 +6,11 @@
 import io.jenkins.plugins.artifact_manager_jclouds.JCloudsArtifactManagerFactory;
 import java.util.logging.Logger;
 import jenkins.model.ArtifactManagerConfiguration;
+
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
```

Comment:
Fine though we conventionally import these things via `Matchers`.

Reply:
https://hamcrest.org/JavaHamcrest/javadoc/2.1/org/hamcrest/Matchers.html#instanceOf-java.lang.Class- in this example

Reply:
Oh ok :) I fixed that, thanks a lot for the reference :)